### DISCUSSION TITLE
Is there a way to make nebula automatically connect to a site?

### DISCUSSION THREAD

**Discussion by account-for-school** [NebulaServices/Nebula#51] [General]
I hope there is

**Comment by ghost** [comment]
If you are talking about automatically without typing the link, this is helpful:
1. Type in the link on a nebula proxy
2. Right-click to get options
3. Click on 'View Frame Source' right under 'View Page Source'
4. Copy the link, but remove the 'view-source:' part
5. Paste the link without the view-source: text.
6. Finished

**Reply by cloverdata** [reply]
Yeah, or while in devtools use the built in encoder, eg: `__uv$config.encodeUrl("google.com")`
